src/library/fixitem/FixItemUI.java

```diff
@@ -4,97 +4,99 @@
 
 public class FixItemUI {
 
-    private enum uI_sTaTe { INITIALISED, READY, INSPECTING, COMPLETED };
+    private enum FixItemUIState { INITIALISED, READY, INSPECTING, COMPLETED };
 
-    private FixItemControl CoNtRoL;
-    private Scanner InPuT;
-    private uI_sTaTe StAtE;
+    private FixItemControl control;
+    private Scanner scanner;
+    private FixItemUIState uiState;
 
     
-    public FixItemUI(FixItemControl CoNtRoL) {
-        this.CoNtRoL = CoNtRoL;
-        InPuT = new Scanner(System.in);
-        StAtE = uI_sTaTe.INITIALISED;
-        CoNtRoL.setUI(this);
+    public FixItemUI(FixItemControl control) {
+        this.control = control;
+        scanner = new Scanner(System.in);
+        uiState = FixItemUIState.INITIALISED;
+        control.setUI(this);
     }
 
 
-    public void RuN() {
-        DiSpLaY_OuTpUt("Fix Item Use Case UI\n");
+    public void run() {
+        displayOutput("Fix Item Use Case UI\n");
         
         while (true) {
             
-            switch (StAtE) {
+            switch (uiState) {
             
             case READY:
-                String ITem_EnTrY_StRiNg = GeTiNpUt("Scan Item (<enter> completes): ");
-                if (ITem_EnTrY_StRiNg.length() == 0) 
-                    CoNtRoL.processingCompleted();
-                
+                String itemEntryString = getInput("Scan Item (<enter> completes): ");
+                if (itemEntryString.length() == 0) {
+                    control.processingCompleted();
+                }
+
                 else {
                     try {
-                        long itEM_Id = Long.valueOf(ITem_EnTrY_StRiNg).longValue();
-                        CoNtRoL.itemScanned(itEM_Id);
+                        long itEM_Id = Long.valueOf(itemEntryString).longValue();
```

Comment:
PR 47 D1 CSG#2 All variable names are to start with lowercase letter and to be in camelBack

Reply:
[2d2b5d8](https://github.com/elliotmitchum/ITC205/pull/47/commits/2d2b5d82544fa09692d3ac50aa7bf92cb98dc016)